backend/src/main/java/gov/cdc/usds/simplereport/config/authorization/DemoAuthenticationConfiguration.java

```diff
@@ -122,11 +126,20 @@ public static IdentitySupplier getCurrentDemoUserSupplier(DemoUserConfiguration
 
     private final IdentitySupplier _getCurrentUser;
     private final OktaRepository _oktaRepo;
+    private final Set<String> adminGroupMemberSet;
```

Comment:
I know we have different conventions in different classes (the price of not ever having arguments about naming conventions) but please follow the local convention here.
```suggestion
    private final Set<String> _adminGroupMemberSet;
```